tsl/src/hypercore/hypercore_handler.c

```diff
@@ -1787,6 +1789,87 @@ hypercore_tuple_delete(Relation relation, ItemPointer tid, CommandId cid, Snapsh
 	return result;
 }
 
+/*
+ * Decompress a segment that contains the row given by ctid.
+ *
+ * This function is typically called during an upsert (ON CONFLICT DO UPDATE),
```

Comment:
Nit: it is only called in this situation.
```suggestion
 * This function is called during an upsert (ON CONFLICT DO UPDATE),
```